packages/node/src/methods/state-channel/withdraw/controller.ts

```diff
@@ -1,8 +1,10 @@
+import { xkeyKthAddress } from "@counterfactual/machine";
```

Comment:
`Kth`?

Reply:
The `k`-th child of the Node's [extended key](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#extended-keys)